src/components/ProjectListItemCard/ProjectListItemCard.tsx

```diff
@@ -0,0 +1,169 @@
+import {
+  Box,
+  Card,
+  CardContent,
+  CardMedia,
+  Chip,
+  makeStyles,
+  Typography,
+} from '@material-ui/core';
+import { VerifiedUserOutlined } from '@material-ui/icons';
+import { FC } from 'react';
+import * as React from 'react';
+import { ProjectStatus, Sensitivity } from '../../api';
+
+const useStyles = makeStyles({
```

Comment:
These styles look good. For the `padding` & `margin` though let's use the `spacing` function from the theme.
It keeps everything on an 8px grid and is a bit easier for us with those rails.

Check out some examples in [LoginForm](https://github.com/SeedCompany/cord-field/blob/v3/src/scenes/Authentication/Login/LoginForm.tsx#L31-L69)

Reply:
Also I know these won't match figma exactly, so round to the nearest value.